pyproject.toml

```diff
@@ -0,0 +1,152 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "kagglehub"
+dynamic = [
+    "version"
+]
+description = "Access Kaggle resources anywhere"
+authors = [
+    { name = "Kaggle", email = "support@kaggle.com" },
+]
+license = { file = "LICENSE" }
+readme = "README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+]
+keywords = ["Kaggle", "API"]
+requires-python = ">=3.7"
+dependencies = []
```

Comment:
Will have a dependency on `kagglesdk` which is the new auto-generated low-level Python client.